src/configuration/routes/cache.md

```diff
@@ -85,96 +126,98 @@ cache:
+### Cache per route
 
-For example, if the `headers` key is the following:
+If you need fine-grained caching, you can set up caching rules for several routes separately:
 
 ```yaml
-cache:
-  enabled: true
-  headers: ["Accept"]
+https://{default}/:
+  type: upstream
+  upstream: app:http
+  cache:
+    enabled: true
+
+https://{default}/foo/:
+  type: upstream
+  upstream: app:http
+  cache:
+    enabled: false
+
+https://{default}/foo/bar/:
+  type: upstream
+  upstream: app:http
+  cache:
+    enabled: true
 ```
 
-Then Platform.sh will cache a different response for each value of the
-`Accept` HTTP request header.
+With this configuration, the following routes are cached:
 
-> **note**
-> The following request headers cannot be used as cache key:
-> `Accept-Encoding`, `Connection`, `Proxy-Authorization`, `TE`, `Upgrade`.
+-   `https://{default}/`
+-   `https://{default}/foo/bar/`
+-   `https://{default}/foo/bar/baz/`
 
-### `cookies`
+And the following routes are **not** cached:
 
-The `cookies` key define on which values the cache key must depend.
+-   `https://{default}/foo/`
+-   `https://{default}/foo/baz/`
 
-For example:
+> **note**
+> Regular expressions in routes are **not** supported.
+
+### Allowing only specific cookies
+
+Some applications use cookies to invalidate cache responses, but expect other cookies to be ignored. This is a simple case of allowing only a subset of cookies.
 
 ```yaml
 cache:
   enabled: true
-  cookies: ["foo"]
+  cookies: ["MYCOOKIE"]
 ```
-
-The cache key will depend on the value of the `foo` cookie in the
-request.
-
-A special case exists if the `cookies` key has the `["*"]` value: it
-means that any request with a cookie will bypass the cache. This is the
-default value.
-
 > **note**
-> You can not  use wildcards in the cookie name, either use a precise cookie
+> You can not use wildcards in the cookie name. Either use a precise cookie
 > name, or match all cookies with a `"*"`. `"SESS*"` or `"~SESS"` are currently
 > not valid values.
 
-### `default_ttl`
+### Cache HTTP and HTTPS separately using the `Vary` header
+
+Set the Vary header to `X-Forwarded-Proto` [custom request header](/development/faq.md#do-you-add-custom-http-headers) to render content based on the request protocol (i.e. HTTP or HTTPS). By adding `Vary: X-Forwarded-Proto` to the response header, HTTP and HTTPS content would be cached separately.
 
-If the response does not have a `Cache-Control` header, the
-`default_ttl` key is used to define the cache duration, in seconds. The
-default value is `0`.
+### Cache zipped content separately
 
-If the application code returns a Cache-Control header, or if your `.platform.app.yaml` file is configured to set a cache lifetime for static assets, then this value is ignored in favor of the application container's headers.
+Use `Vary: Accept-Encoding` to serve different content depending on the encoding. Useful for ensuring that gzipped content is not served to clients that can read it.
```

Comment:
Shouldn't there be another negative in there?  We don't want to make sure non-gzip clients only get gzip content, I assume...